src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,9 +48,24 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+
+        final ArrayList<String> lower_keywords = new ArrayList<>();
+        //get a lowercase keywords list
+        for(String newSplitWords : keywords)
+        {
```

Comment:
I think you should switch to the "Egyptian" style brackets as mentioned in the Java Coding Standards for the module. The rationale given by the Prof is "just follow it".